README.md

```diff
@@ -35,25 +34,44 @@ You need [Google Chrome](https://www.google.com/chrome/browser/desktop/). *not t
 
 <img src="https://github.com/isaacbernat/netflix-to-srt/blob/master/chrome_console.png?raw=true" alt="Chrome console screenshot" width="557px" height="607px">
 
-### From Netflix: method 3
+## Method 3
 The information is extracted from [this post](http://forum.opensubtitles.org/viewtopic.php?t=15141).
 
 You need FireFox and AdblockPlus Add-On. *not tested on other browsers*
 - Start Netflix and your movie/episode (stream is active!)
 - Start AdblockPlus, open blockable items
-- Search: dfxp *(e.g. >> #.nflximg.com/#/#/########.dfxp?v=1&e=#########&t=######_#####&random=1234567890)*
+- Search: dfxp *(e.g. `>> #.nflximg.com/#/#/########.dfxp?v=1&e=#########&t=######_#####&random=1234567890`)*
 - open the dfxp in a new window
 - Save as
 
-### From YouTube
+# 2. YouTube-To-SRT
```

Comment:
Unless you think the previous heading was not clear enough, this would benefit a lot from SEO perspective or similar reasons, I think It'd keep the existing one.

Reply:
ok